### PR TITLE
Fix lint issues

### PR DESCRIPTION
**Pull request by sylviamoss**

### CODE REVIEW

File: builder/triton/access_config.go [hunk 1]
```diff
@@ -180,7 +179,7 @@ func (c *Client) Compute() (*compute.ComputeClient, error) {
 func (c *Client) Network() (*network.NetworkClient, error) {
 	networkClient, err := network.NewClient(c.config)
 	if err != nil {
-		return nil, errwrap.Wrapf("Error Creating Triton Network Client: {{err}}", err)
```
Comment (azr) [inline builder/triton/access_config.go:183]: Hm, was that an okay error wrapping ? 🤔

Comment (azr) [reply builder/triton/access_config.go:183]: def LGTM: https://github.com/hashicorp/errwrap

Comment (sylviamoss) [reply builder/triton/access_config.go:183]: Maybe 🤔  the linter suggested replacing with fmt.Errorf because this one is deprecated

Comment (azr) [reply builder/triton/access_config.go:183]: ah no the linter is right !

**Comment by azr** [comment]
👍🏼

**Pull request merged**